docs/hadoop-catalog.md

```diff
@@ -35,6 +35,22 @@ Besides the [common catalog properties](./gravitino-server-config.md#gravitino-c
 | `authentication.kerberos.check-interval-sec`       | The check interval of Kerberos credential for Hadoop catalog.                                  | 60            | No                                                          | 0.5.1         |
 | `authentication.kerberos.keytab-fetch-timeout-sec` | The fetch timeout of retrieving Kerberos keytab from `authentication.kerberos.keytab-uri`.     | 60            | No                                                          | 0.5.1         |
 
+
+### Authentication Strategy for Hadoop Catalog
+
+The Hadoop catalog supports multi-level authentication to control access, allowing different strategies for the catalog, schema, and fileset. The priority of authentication strategies is as follows: catalog < schema < fileset.
```

Comment:
First. Is it a strategy? I have some concern about this term. They are different mechinisms for me instead of different straties. 

Second. The parent strategy will override the child or the parent property will override the children one. They are different.

Reply:
> First. Is it a strategy? I have some concern about this term. They are different mechinisms for me instead of different straties.

I will check the difference between them and choose a better one.

> Second. The parent strategy will override the child or the parent property will override the children one. They are different.

Let me think about it.